ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -515,17 +522,29 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         tap(() => busyCounter$.next(-1))
     );
 
+    const canSaveRealAddress = (address: Address | null) =>
+        this.enoughDataToSave(address)  && this.meetsActivityRequirements(address);
+
     // "Real" as opposed to "Temp"
     const saveRealAddressAction$ = this.saveTrigger$.pipe(
         withLatestFrom(currentAddress$),
-        filter(([_, addressToSave]) => this.enoughDataToSave(addressToSave)),
+        filter(([_, addressToSave]) => canSaveRealAddress(addressToSave)),
         tap(() => busyCounter$.next(1)),
         concatMap(([_, addressToSave]) => this.addressService.saveAddress(addressToSave, false)),
         removeTempAddressOperator(),
         tap(() => busyCounter$.next(-1)),
         share()
     );
 
+    /* If we get to show validations, we touch controls so we will show validations errors
+        associate with controls
+     */
+    const touchFormOnSubmitWithBadAddressAction$ = this.validationRequested$.pipe(
+        withLatestFrom(currentAddress$),
+        filter(([_, addressToSave]) => !canSaveRealAddress(addressToSave)),
+        tap(() => this.addressInputComponent.touchAllControls())
+    );
+
```

Comment:
This makes sure that if phone is required and the field is empty, the validation error associated with control is displayed.